_includes/scenarios/integrate/updating-identityattributes-of-yourself.md

```diff
@@ -1,17 +1,45 @@
 
-Consider an IdentityA which shared its DisplayName to IdentityB, IdentityC, and IdentityD. IdentityA now would like to change the DisplayName and has to execute the following steps:
+In contrast to [RelationshipAttributes]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) you can [create IdentityAttributes]({% link _docs_use-cases/use-case-consumption-create-an-attribute.md %}) privately without sharing them with anyone.
+In order to [share an IdentityAttribute]({% link _docs_use-cases/use-case-consumption-share-an-identityattribute.md %}), you need to send an according [Request]({% link _docs_integrate/data-model-overview.md %}#request) to the peer.
+If they accept, a new [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute) will be created at the peer's side.
+It has the same content like your private IdentityAttribute and, in addition, a `shareInfo` property.
+It stores information about the respective `peer`, the `sourceReference` to the private IdentityAttribute, whose content you copied, and a reference to the Request used to share the Attribute.
+Moreover, another copy will be created at your side, that has a `shareInfo`, too.
+Thus, sharing an IdentityAttribute will create an own shared IdentityAttribute copy for every peer you shared the Attribute with at your side and a peer shared IdentityAttribute copy for each peer at their side.
 
-- Succeed the LocalAttribute of the current DisplayName with the new DisplayName
-- Notify each peer individually that the DisplayName changed (this is optional, for every peer it can be decided if the peer receives the new Attribute or not)
+Now, let's consider the case that the value of your IdentityAttribute changes and you want to update it.
+To [succeed the (unshared) IdentityAttribute]({% link _docs_use-cases/use-case-consumption-succeed-an-identityattribute.md %}), a new version with the updated value will be created.
+In its `succeeds` property the predecessor is stored.
+It in turn is updated, too, to link to the newer version in its `succeededBy` property.
+Consequently, succeeding an Attribute, a double-linked list of all versions is created.
 
-Each peer would receive a Notification containing an AttributeSucceededNotificationItem which is automatically processed by each Identity. Thus, succeeding Attribute does not result into decidable Requests for the peer. They are only informed about the change.
+Next, you can [notify your peers about the succession]({% link _docs_use-cases/use-case-consumption-notify-peer-about-identityattribute-succession.md %}), so that they'll receive the current version, too.
+Note that it is also possible to make a selection of the peers you want to notify about the Attribute succession, if for some reason you don't want to inform all of them.
+The own shared IdentityAttributes associated with the peers chosen will be succeeded in the same manner than your unshared IdentityAttribute before.
```

Comment:
_The own shared IdentityAttributes **shared** with the **chosen peers** will be succeeded in the same manner **as** your unshared IdentityAttribute._

- associated is unnecessarily generic